packages/tokens/properties/size/font.json

```diff
@@ -1,44 +1,68 @@
 {
-  "legacy": {
+  "eds": {
     "size": {
       "font": {
-        "xl": {
-          "value": "1.5rem",
+        "h1": {
+          "value": "3rem",
           "comment": "Font size for Heading 1 sized text"
         },
-        "lg": {
-          "value": "1.125rem",
+        "h2": {
+          "value": "2.5rem",
           "comment": "Font size for Heading 2 sized text"
         },
-        "md": {
-          "value": "1rem",
-          "comment": "Font size for medium text"
+        "h3": {
+          "value": "2rem",
+          "comment": "Font size for Heading 3 sized text"
         },
-        "sm": {
-          "value": "0.875rem",
-          "comment": "Font size for small text"
+        "h4": {
+          "value": "1.5rem",
+          "comment": "Font size for Heading 4 sized text"
+        },
+        "h5": {
+          "value": "1.125rem",
+          "comment": "Font size for Heading 5 sized text"
         },
         "base": {
```

Comment:
we can leave base since we'll use it in the root/body to set 1rem = 16px, but we should probably also have a body value.